jobs/apiserver/templates/drain-script.erb

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+RUN_DIR=/var/vcap/sys/run/apiserver
+LOG_DIR=/var/vcap/sys/log/apiserver
+LOGFILE=$LOG_DIR/drain.log
+PIDFILE=$RUN_DIR/apiserver.pid
+
+exec 3>&1
+
+exec 1>> $LOGFILE
+exec 2>> $LOGFILE
+
+if [ ! -f $PIDFILE ]; then
```

Comment:
This isn't your fault but I commented on Bo's PR which also needed the common/utils.sh functionality.
Let us just copy over the cf-release/src/common package and use the functions provided there to manage PID files and Drain.
I don't want to have this done in 5 different files.